.gitignore

```diff
@@ -49,7 +49,6 @@ cypress/results
 .sentryclirc
 
 # Version control
-yarn.lock

```

Comment:
Added yarn.lock back to vc, we should really be deploying with the same versions